### PR TITLE
Add support for Google Maps Map ID prop

### PR DESCRIPTION
**Pull request by andreeaceachir142**
# What 

- Add support for Google Maps Map ID prop

# How 

- Pass the new `gmMapId` prop to the `GoogleMapsMap.jsx` inside the `mapViewOptions` object

### CODE REVIEW

File: packages/map-template/src/components/Map/GoogleMapsMap/GoogleMapsMap.jsx [hunk 1]
```diff
@@ -39,7 +41,8 @@ function GoogleMapsMap({ onMapView, onPositionControl }) {
                 // Always set a center so the map so the bounds or center can be read from the start.
                 center: { lat: 0, lng: 0 },
                 // Set a large zoom so we prevent a "zoom 0 glitch" (showing the whole globe temporarily)
-                zoom: 21
+                zoom: 21, 
+                mapId: gmMapId
```
Comment (ammapspeople) [inline packages/map-template/src/components/Map/GoogleMapsMap/GoogleMapsMap.jsx:45]: Would it be more safe to exclude completely if not set? Does it work as expected when passing the default `null`?

File: packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx [hunk 2]
```diff
@@ -19,6 +19,7 @@ import defaultLogo from "../../assets/logo.svg";
  * @param {array} [props.externalIDs] - Filter locations shown on the map based on the external IDs.
  * @param {string} [props.tileStyle] - Tile style name to change the interface of the map.
  * @param {number} [props.startZoomLevel] - The initial zoom level of the map.
+ * @param {string} [props.gmMapId] - The map ID associated with a specific map style or feature.
```
Comment (ammapspeople) [inline packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx:22]: We should probably specify that it is a Google Maps Map ID.

Comment (andreeaceachir142) [reply packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx:22]: Very good point. I'll specify that.

File: packages/map-template/src/components/Map/GoogleMapsMap/GoogleMapsMap.jsx (hunk 1 above)
Comment (ammapspeople) [reply packages/map-template/src/components/Map/GoogleMapsMap/GoogleMapsMap.jsx:45]: The atom has `null` as default. If that works, then no need to change anything 😄

Comment (andreeaceachir142) [reply packages/map-template/src/components/Map/GoogleMapsMap/GoogleMapsMap.jsx:45]: Yeah, it works 👍

**Review by ammapspeople — APPROVED** [review]
👏

**Pull request merged**